build.gradle

```diff
@@ -19,6 +19,19 @@ tasks.register('project_version') {
     }
 }
 
+def releaseInfoTask = tasks.register('releaseInfo') {
```

Comment:
This gradle tasks produced a JSON file containing info about a release build. Names?

1. releaseInfo
2. releaseJson
3. releaseInfoJson

The thought is to establish this `release.json` file in Release assets/artifacts as a pattern across several project repositories to help automated updates of references to "the latest version of X"